src/components/date-picker/date-picker.scss

```diff
@@ -1,233 +1,13 @@
-    }
-
-    .flatpickr-current-month {
-        div.flatpickr-month {
-            font-size: pxToRem(16);
-            font-weight: 400 !important;
-            color: $datepickerTextOnThemeColor;
-        }
-
-        select {
-            color: $datepickerTextOnThemeColor;
-        }
-
-        input {
-            color: $datepickerTextOnThemeColor;
-        }
-
-        .numInputWrapper:hover {
-            background: $datepickerHeaderColor;
-        }
-    }
-
-    .flatpickr-weekday {
-        background: $datepickerHeaderColor;
-        color: $datepickerTextOnThemeColor;
-        font-weight: 400;
-    }
-
-    .flatpickr-weekdays {
-        background: $datepickerHeaderColor;
-    }
-
-    // Hide the arrow between the input box and the calendar
-    &.arrowTop:before,
-    &.arrowTop:after {
-        display: none;
-    }
-
-    .flatpickr-day {
-        border: none;
-        background: transparent !important;
-        border-radius: 0 !important;
-        box-shadow: none !important;
-        max-width: none;
-
-        &.selected,
-        &.inRange,
-        &.selected:hover,
-        &.nextMonthDay:hover,
-        &:hover {
-            background: transparent;
-            color: $datepickerTextOnThemeColor;
-        }
-        &.today {
-            background: transparent;
-            &:after {
-                background: var(
-                    --mdc-theme-text-hint-on-light,
-                    $mdc-theme-text-hint-on-light
-                );
-                opacity: 0.25;
-            }
-            &:hover {
-                background: transparent;
-                color: $datepickerTextOnThemeColor;
-            }
-        }
-
-        // The circle around the date should be smaller than
-        // the original, but it's hard to actually style
-        // the original one and still keep some of the space
-        // between the days from our design.
-        // So the original circle is hidden and a new one
-        // is created using a pseudo element.
-        &:after {
-            border-radius: pxToRem(15);
-            content: ' ';
-            height: pxToRem(30);
-            left: 50%;
-            margin-left: pxToRem(-15);
-            margin-top: pxToRem(-15);
-            position: absolute;
-            top: 50%;
-            width: pxToRem(30);
-            z-index: -1;
-        }
-
-        &.selected {
-            &:after {
-                background: $datepickerSelectedColor;
-                color: $datepickerTextOnThemeColor;
-            }
-        }
-
-        &.inRange,
-        &.nextMonthDay:hover,
-        &:hover {
-            &:after {
-                background: $datepickerHoverHighlightColor;
-                color: $datepickerTextOnThemeColor;
-                opacity: 0.5;
-            }
-        }
-    }
-
-    // The column with week numbers (including the title)
-    .flatpickr-weekwrapper {
-        width: pxToRem(40);
-    }
+div.container {
+    position: relative;
```

Comment:
Not sure if this is needed either, if the position styling of limel-portal isn't needed.

Reply:
This is indeed needed, otherwise the calender is not shown at all